### PR TITLE
PendingReleaseNotes: add a note about telemetry leaderboard

### PR DESCRIPTION
**Pull request by yaarith**
## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: PendingReleaseNotes [hunk 1]
```diff
@@ -124,6 +124,14 @@
 * RBD: list-watchers C++ API (`Image::list_watchers`) now clears the passed
   `std::list` before potentially appending to it, aligning with the semantics
   of the corresponding C API (`rbd_watchers_list`).
+* Telemetry: Users who are opted-in to telemetry can also opt-in to
+  participating in a leaderboard in the telemetry public
+  dashboards (https://telemetry-public.ceph.com/). Users can now also add a
```
Comment (ljflores) [inline PendingReleaseNotes:129]: Space after "dashboards"
```suggestion
  dashboards  (https://telemetry-public.ceph.com/). Users can now also add a
```

**Review by ljflores — APPROVED** [review]

**Review by neha-ojha — APPROVED** [review]

File: PendingReleaseNotes [hunk 2]
```diff
@@ -124,6 +124,14 @@
...
   of the corresponding C API (`rbd_watchers_list`).
+* Telemetry: Users who are opted-in to telemetry can also opt-in to
+  participating in a leaderboard in the telemetry public
+  dashboards
(https://telemetry-public.ceph.com/). Users can now also add a
+  description of the cluster to publicly appear in the leaderboard.
+  For more details, see:
+  https://docs.ceph.com/en/latest/mgr/telemetry/#leaderboard
+  See a sample report with `ceph telemetry preview`.
+  Opt-in with `ceph telemetry on`.
```
Comment (batrick) [inline PendingReleaseNotes:134]:
```suggestion
  Opt-in with `ceph config set mgr mgr/telemetry/leaderboard true`.
```

?

**Review by batrick — CHANGES_REQUESTED** [review]

File: PendingReleaseNotes (hunk 1 above)
Comment (yaarith) [reply PendingReleaseNotes]: @ljflores thanks for catching this, there was an issue with encoding

File: PendingReleaseNotes (hunk 2 above)
Comment (yaarith) [reply PendingReleaseNotes]: @batrick thanks for the suggestion; I clarified the use of the commands

**Review by batrick — APPROVED** [review]

**Comment by yaarith** [comment]
jenkins test make check

**Comment by yaarith** [comment]
jenkins test make check

**Comment by yaarith** [comment]
jenkins retest this please

**Pull request merged**